### PR TITLE
Improves the module examples/complete

### PR DESCRIPTION
**Pull request by MrWolong**

### CODE REVIEW

File: README-CN.md [hunk 1]
```diff
@@ -42,8 +37,77 @@ module "kms"  {
...
+  region      = "cn-beijing"
+  profile     = "Your-Profile-Name"
+  description             = "Hello KMS"
+  deletion_window_in_days = "7"
+  is_enabled              = true
+  // ...
+}
+```
+
+如果你想对正在使用中的Module升级到 1.1.0 或者更高的版本，那么你可以在模板中显示定义一个系统过Region的provider：
```
Comment (xiaozhu36) [inline README-CN.md]: 系统过 -》 相同

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

File: examples/complete/README.md [hunk 2]
```diff
@@ -12,6 +12,21 @@ $ terraform plan
 $ terraform apply
 ```
 
-Note that this example will create resources which cost money. Run `terraform destroy` when you don't need these resources.
+Note that this example may create resources which cost money. Run `terraform destroy` when you don't need these resources.
```
Comment (Wanghx0991) [inline examples/complete/README.md:15]: markdown的格式统一起来， 这里跟ram-fc的module保持一致

File: variables.tf [hunk 3]
```diff
@@ -1,26 +1,38 @@
...
+
+variable "is_enabled" {
+  description = "(Deprecated from version 1.2.0) Specifies whether the key is enabled. Default to true.New field 'status' instead."
+  type        = bool
+  default     = true
+}
+
 #kms key
 variable "create_kms" {
   description = "Whether to create kms key resource. Default to true."
```
Comment (Wanghx0991) [inline variables.tf:38]: type

File: main.tf [hunk 4]
```diff
@@ -1,24 +1,15 @@
...
-  count                   = var.use_existing_key == true ? 0 : var.create_kms ? 1 : 0
-  description             = var.description
-  key_usage               = var.key_usage
-  deletion_window_in_days = var.deletion_window_in_days
-  is_enabled              = var.is_enabled
+  count                  = var.use_existing_key == true ? 0 : var.create_kms ? 1 : 0
+  description            = var.description
+  key_usage              = var.key_usage
+  pending_window_in_days = var.pending_window_in_days
+  status                 = var.status
```
Comment (Wanghx0991) [inline main.tf:12]: is_enabled - (Optional) Field is_enabled has been deprecated from provider version 1.85.0. New field key_state instead.

这里应该是`key_state` 吧

File: variables.tf [hunk 5]
```diff
@@ -46,16 +58,16 @@ variable "key_usage" {
...
   type        = string
-  default     = 30
+  default     = "30"
 }
 
-variable "is_enabled" {
-  description = "Specifies whether the key is enabled. Default to true."
-  type        = bool
-  default     = true
+variable "status" {
```
Comment (Wanghx0991) [inline variables.tf:67]: 确认下这块

File: variables.tf [hunk 6]
```diff
@@ -1,26 +1,38 @@
...
 
 variable "skip_region_validation" {
-  description = "Skip staticvalidation of region ID. Used by users of alternative AlibabaCloud-like APIs or users w/ access to regions that are not public (yet)."
+  description = "(Deprecated from version 1.1.0) Skip staticvalidation of region ID. Used by users of alternative AlibabaCloud-like APIs or users w/ access to regions that are not public (yet)."
   type        = bool
   default     = false
 }
 
+variable "deletion_window_in_days" {
+  description = "(Deprecated from version 1.2.0) Duration in days after which the key is deleted after destruction of the resource, must be between 7 and 30 days. Default to 30 days.New field 'pending_window_in_days' instead."
```
Comment (Wanghx0991) [inline variables.tf]: 当前版本是V1.2.0， 字段废弃应该从本周的V1.3.0开始吧

File: examples/complete/README.md (hunk 2 above)
Comment (MrWolong) [reply examples/complete/README.md:15]: ram-fc就是这样的

File: main.tf (hunk 4 above)
Comment (MrWolong) [reply main.tf:12]: key_state - (Optional) Field key_state has been deprecated from provider version 1.123.1. New field status instead.

File: variables.tf (hunk 5 above)
Comment (MrWolong) [reply variables.tf:67]: The status of CMK. Defaults to Enabled. Valid Values: Disabled, Enabled, PendingDeletion.

**Pull request closed without merging**